### PR TITLE
Add stats to profile and various 💄

### PR DESCRIPTION
**Pull request by bjarkehs**
image

Fixes layout on player page for teammates and opponents
image

### CODE REVIEW

File: frontend/src/routes/(authed)/player/[id]/+page.svelte [hunk 1]
```diff
@@ -84,10 +84,12 @@
...
   {/if}
 
-  {#if data.stats.opponents.length > 0}
+  {#if data.opponents.length > 0}
     <Card.Root>
       <Card.Content class="flex flex-col p-0 md:p-6">
-        <h2 class="text-xl md:text-2xl">Most common opponents</h2>
+        <h2 class="px-4 py-3 text-xl md:p-0 md:text-2xl">
+          ⚔️ Most common opponents
+        </h2>
```
Comment (lajtmaN) [inline frontend/src/routes/(authed)/player/[id]/+page.svelte]: We should properly soon make a Header component. We have this kind of h2 lots of places 😅

**Review by lajtmaN — COMMENTED** [review]
Instead of having a profile page with stats (which is duplicated), could we then update the player page such that the title indicates that it is "you" and show the logout button next to it, and then all the remaining stuff. 

When you navigate to profile page, and the profile is attached to a player, then redirect. Otherwise show the claim functionality

**Comment by bjarkehs** [comment]
> Instead of having a profile page with stats (which is duplicated), could we then update the player page such that the title indicates that it is "you" and show the logout button next to it, and then all the remaining stuff.
> 
> When you navigate to profile page, and the profile is attached to a player, then redirect. Otherwise show the claim functionality

I think ideally it would be two different routes with the same content. But I'm not sure if that's easy to do in SvelteKit. So I think your idea is the most feasible approach currently. I'll look into it :+1:

**Comment by bjarkehs** [comment]
image


image

**Comment by bjarkehs** [comment]
Only issue is that the button doesn't highlight in the navbar when we redirect.

But this could be an issue we solve later

File: frontend/src/routes/(authed)/player/[id]/+page.svelte (hunk 1 above)
Comment (bjarkehs) [reply frontend/src/routes/(authed)/player/[id]/+page.svelte]: Yep 😅

**Review by lajtmaN — APPROVED** [review]

**Pull request merged**